prisma/schema.prisma

```diff
@@ -77,3 +80,51 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+model Volunteer {
+    user   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
+    userId String @id
+
+    activities Activity[]
+}
+
+model Administrator {
+    user   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
+    userId String @id
+
+    Activity Activity[]
+}
+
+model Activity {
+    id             String   @id @default(cuid())
+    title          String
+    description    String
+    startTimestamp DateTime
+    endTimestamp   DateTime
+    isVisible      Boolean
+
+    tags     Tag[]
+    capacity Int
+
+    volunteers      Volunteer[]
+    manager         Administrator @relation(fields: [administratorId], references: [userId])
+    administratorId String
+
+    primaryContactInfo String
+    organization       NPO    @relation(fields: [npoId], references: [id])
+    npoId              String
+}
+
+model Tag {
+    id         String     @id @default(cuid())
+    activities Activity[]
+}
+
+model NPO {
```

Comment:
i think can add NPO field `link` or `website-url` cos its safe to assume every NPO should have a website up right, so can even make it compulsory field